jp/components/calendar.md

```diff
@@ -387,16 +388,16 @@ Result:
 | 名前   | 引数 | 戻り値の型 | 説明 |
 |:----------:|:------|:------|:------|
 | `selectDate` | `date: Date` または `Date[]` | `void` | カレンダーの選択を変更します。このメソッドの呼び出しは `onSelection` イベントを発生させます。 |
```

Comment:
Unfortunately I'm having the same problem here.  Lines 374 and 375 are in English because they were just added.  Could you please localize them?  I can't write a comment on those lines!